docs/sources/awsiot.md

```diff
@@ -2,6 +2,10 @@
 
 ![TriggerMesh Knative Lambda Sources](./images/klass.png "TriggerMesh Knative Lambda Sources")
 
+## Prerequisite(s)
+
+- Knative Lambda Sources (KLASS)
+
```

Comment:
In fact this file `awsiot.md` is not linked and not used.
I think we could remove it.

@sebgoa @antoineco ^^

Reply:
We can and should, yes.

Reply:
@odacremolbap are you meaning to delete the whole file?

Reply:
@odacremolbap I can delete the files if that's what you would like?

Reply:
yes delete please

Reply:
I will delete the files